### PR TITLE
feat: add Strategy[StrategyCurvemUSDVoterProxy]

### PR DESCRIPTION
**Pull request by orbxball**

### CODE REVIEW

**Review by storming0x — APPROVED** [review]
Diffed against [StrategyCurve3CrvVoterProxy](https://github.com/iearn-finance/yearn-protocol/blob/develop/contracts/strategies/StrategyCurve3CrvVoterProxy.sol) and LGTM, applied all house keeping changes.

File: contracts/strategies/StrategyCurvemUSDVoterProxy.sol [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+        require(msg.sender == governance, "!governance");
+        strategistReward = _strategistReward;
+    }
+
+    function setProxy(address _proxy) external {
+        require(msg.sender == governance, "!governance");
+        proxy = _proxy;
+    }
+
+    function switchDex() external {
```
Comment (fubuloubu) [inline contracts/strategies/StrategyCurvemUSDVoterProxy.sol]: Same comment about being explicit about the switch as other PR

**Review by fubuloubu — APPROVED** [review]

File: contracts/strategies/StrategyCurvemUSDVoterProxy.sol [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+    function setProxy(address _proxy) external {
+        require(msg.sender == governance, "!governance");
+        proxy = _proxy;
+    }
+
+    function switchDex(bool isUniswap) external {
+        require(msg.sender == strategist || msg.sender == governance, "!authorized");
+        // save gas and storage
+        if (isUniswap) {
+            dex = address(0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D);
```
Comment (fubuloubu) [inline contracts/strategies/StrategyCurvemUSDVoterProxy.sol]: Could these be constants instead?

**Review by storming0x — APPROVED** [review]
LGTM after latest changes

**Review by fubuloubu — APPROVED** [review]

**Review by lbertenasco — APPROVED** [review]

**Review by banteg — APPROVED** [review]

**Pull request merged**